apps/docs/app/ui/layout/wrapper/Wrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import type { ReactNode } from "react";
+import clsx from "clsx";
+
+import "./wrapper.css";
+
+interface WrapperProps {
+    children: ReactNode;
+    type?: "with-sidebar" | "fluid";
+    className?: string;
+}
+
+const Wrapper = ({ children, type = "fluid", className }: WrapperProps) => {
```

Comment:
could we find a less generic name? 

ContentWrapper? ContentLayout?

Reply:
It's generic because it's used in several places on the site, not just in the layout. It can be found, for example, in the navigation and also in the header.